app/main.py

```diff
@@ -1,2 +1,51 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self,
+                 initial_capacity: int = 8,
+                 load_factor: float = 2 / 3) -> None:
+        self.capacity = initial_capacity
+        self.load_factor = load_factor
+        self.size = 0
+        self.buckets = [[] for _ in range(self.capacity)]
+
+    def __setitem__(self, key: Any, value: Any) -> None:
```

Comment:
`key` should be `Hashable`. Fix everywhere, please
```suggestion
    def __setitem__(self, key: Hashable, value: Any) -> None:
```